src/filters/convolute_filter.class.js

```diff
@@ -311,7 +311,7 @@
      */
     getUniformLocations: function(gl, program) {
       return {
-        uMatrix: gl.getUniformLocation(program, 'uMatrix'),
+        uMatrix: gl.getUniformLocation(program, 'uMatrix[0]'),
```

Comment:
can you explain me this? because i have no idea what does it do.
I would expect this to be just a reference to how is named the variable in the shader code.

Reply:
Yes it is a reference to how it's named, my understanding is that either referencing it as `'uMatrix'` or as its first element `'uMatrix[0]'` should work but only the second one worked when I was gl in tests; both seem to work when using it normally.